src/o365/graph/commands/o365group/o365group-user-list.spec.ts

```diff
@@ -278,7 +388,7 @@ describe(commands.TEAMS_USER_LIST, () => {
     });
   });
 
-  it('correctly lists all users in a team (debug)', (done) => {
+  it('correctly lists all users in a office365 group (debug)', (done) => {
```

Comment:
Do you mean a graph request failure for each graph call (owner and members call) or a single to catch both?

Reply:
@appieschot , it is up to you! I think both scenarios will do the work since the both requests are very similar. What I usually do is test all the command requests failures and make sure the command fails gracefully when response code is not 200. 

In your case you do one request to get owners and another to get members. They both can be tested in a case of a request failure. This is just to ensure that command will behave properly even after someone else changes your code, but testing even one of both will be sufficient enough I think.